EdgeOfEmpireBot/EdgeOfEmpireBot/BotApp.cs

```diff
@@ -64,27 +63,27 @@ private async Task InitializeBot()
     {
         var token = GetToken();
 
-        this.client.Log += Log;
-        /* TODO: 
+        client.Log += Log;
```

Comment:
this.{insert variable name} is fine in the constructor when you are passing through a variable with the same name however, using it in the context of the actual classes methods is super redundant. Especially when they are private readonly variables. Making them private by default implies that its only available to that class. So adding "this" when calling it seems redundant and also semantically annoying since you arent passing through another variable with the same name through a method argument, and if you were that would be more of an issue. Aside from that the intellisense also acknowledges that its redundant and it should be removed. Which I agree with.